spec/index.bs

```diff
@@ -4050,6 +4055,39 @@ source-map-v3 format.
 Source maps are optional, but serve as a standardized way to support dev-tool
 integration such as source-language debugging. [[SourceMap]]
 
+{{GPUShaderModuleDescriptor/hints}}, if defined, maps an entry point name from
+the shader to a {{GPUShaderModuleCompilationHints}}. No validation is performed with
+any of these {{GPUShaderModuleCompilationHints}}. Implementations should use any
+information present in the {{GPUShaderModuleCompilationHints}} to perform as much
+compilation as is possible within {{GPUDevice/createShaderModule()}}.
+
+Note: Supplying information in {{GPUShaderModuleDescriptor/hints}} does not have any
```

Comment:
If you want to put all of these paragraphs inside one "Note" block, you can use `<div class=note>`. Probably slightly nicer.

Reply:
I actually intended for these to be distinct notes, since each one has its own independent message. Because you used the word "if" I'm going to assume your comment isn't blocking.